### ISSUE TITLE
UI: (13.C) Overview See All Links Insufficient Contrast (CRASM-371)

### ISSUE THREAD

**Issue by hawkishpolicy** [cisagov/XFD#420]
# 🐛 Summary #
Insufficient contrast is provided for “See All” links on Overview page. The color difference of text and its background should have a contrast ratio of at least 4.5:1. Using contrast analyzers, spot test various text and backgrounds throughout the site or application to ensure a contrast ratio of at least 4.5:1. When insufficient contrast issues are found, adjust either one or the other color to meet a minimum of 4.5:1 ratio.

## Expected behavior ##

- Contrast ratio of 4.5:1 or higher.

## Any helpful log output or screenshots ##

![image](https://github.com/cisagov/XFD/assets/123108455/cd260b56-5897-4fda-baac-3530f9e23bbb)
